state/protocol/badger/state.go

```diff
@@ -369,6 +375,34 @@ func (state *State) bootstrapEpoch(root protocol.Snapshot) func(*transaction.Tx)
 	}
 }
 
+// bootstrapSporkInfo bootstraps the protocol state with information about the
+// spork which is used to disambiguate Flow networks.
+func (s *State) bootstrapSporkInfo(root protocol.Snapshot) func(*badger.Txn) error {
```

Comment:
```suggestion
func (state *State) bootstrapSporkInfo(root protocol.Snapshot) func(*badger.Txn) error {
```
receiver names are different.